.drone.yml

```diff
@@ -9,13 +9,79 @@ pipeline:
     commands:
       - make build
 
+  postgres:
+    image: postgres:9.6.15
+    name: postgres
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    detach: true
+    when:
+      event: push
+
+  prisma:
+    image: prismagraphql/prisma:1.34.8
+    name: prisma
+    environment:
+      PRISMA_CONFIG: |
+        port: 4466
+        databases:
+          default:
+            connector: postgres
+            uri: "postgres://postgres:postgres@postgres:5432/postgres"
+            migrations: true
+            rawAccess: true
+    detach: true
+    when:
+      event: push
+
+  # start houston
+  houston:
+    image: astronomerinc/ap-houston-api:dev
+    name: houston
+    environment:
+      NODE_ENV: "development"
+      WEBSERVER__PORT: "8871"
+      PRISMA__ENDPOINT: "http://prisma:4466/houston"
+      PRISMA__DEBUG: "false"
+      ORBIT__PORT: "8080"
+      DATABASE__CONNECTION: "postgres://postgres:postgres@postgres:5432/postgres"
+      PUBLIC_SIGNUPS: "true"
+      EMAIL_CONFIRMATION: "false"
+      EMAIL__ENABLED: "false"
+      JWT__PASSPHRASE: "secretpassword"
+      DEPLOYMENTS__DATABASE__CONNECTION: "postgres://postgres:postgres@postgres:5432/postgres"
+      DEPLOYMENTS__DATABASE__ENABLED: "true"
+      DEPLOYMENTS__LOG_HELM_VALUES: "true"
+      HELM__RELEASE_VERSION: "0.7.5"
+    detach: true
+    when:
+      event: push
+
+  # wait for DB to come online
+  wait-for-db:
+    image: astronomerio/ap-build:0.1.1
+    commands:
+      - while ! nc -w 1 -z postgres 5432; do sleep 1; done
+      - while ! nc -w 1 -z prisma 4466; do sleep 1; done
+      - while ! nc -w 1 -z houston 8871; do sleep 1; done
+    when:
+      event: push
+
   # Test the code
   test:
-    image: asoldatenko/dnd-golang
+    image: asoldatenko/dnd-golang:19.03
+    secrets: [ codecov_token ]
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
yes we need docker cli available, because we have a test where we execute docker command. https://github.com/astronomer/astro-cli/blob/master/docker/docker_test.go#L6-L10

Reply:
The use case for dind is to run docker within docker. It's not intended for running sibiling-style docker containers where a container mounts the socket from the host. You can run docker CLI just fine inside a normal container if docker is installed within and the docker socket is mounted. For example https://github.com/astronomer/ap-build/blob/91dee715aac7239fa1fdcfa55c4838fbac089694/Dockerfile#L25

Reply:
good catch, 
if it's okey i'll change it in separate PR, because we added it not in this PR.